system.properties

```diff
@@ -1 +1,2 @@
-java.runtime.version=17
\ No newline at end of file
+java.runtime.version=17
+springdoc.swagger-ui.enabled=true
```

Comment:
We shouldn't commit this line - or at least it should default to false and we should look at how to enable it for heroku stage and keep it off for heroku prod. Can we use env variables here?